docs/variants/pc_engines/initial-deployment.md

```diff
@@ -0,0 +1,85 @@
+# Initial deployment
+
+This document describes the process of updating
+[PC Engines coreboot firmware](https://pcengines.github.io/) to Dasharo firmware
+based on EDK II.
```

Comment:
I know that both EDK II and UEFI are correct, but we should stick to one, or explain is somewhere why we are using both  (or link to the explanation).